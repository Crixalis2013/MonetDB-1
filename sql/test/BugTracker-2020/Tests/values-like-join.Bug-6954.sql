create procedure profiler.starttrace() external name profiler."starttrace";
create procedure profiler.stoptrace() external name profiler.stoptrace;

call profiler."starttrace"();
select 1 from (values ('one'), ('two')) as l(s), (values ('three'), ('four')) as r(s) where l.s like r.s;
call profiler.stoptrace();
select count(*) from sys.tracelog() where stmt like '%cross%'; -- no crossproducts

drop procedure profiler.starttrace();
drop procedure profiler.stoptrace();

select 1 from (values ('one'), ('two')) as l(s), (values ('three'), ('four')) as r(s) where l.s like r.s;
