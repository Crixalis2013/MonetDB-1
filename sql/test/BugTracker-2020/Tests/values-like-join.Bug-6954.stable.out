stdout of test 'values-like-join.Bug-6954` in directory 'sql/test/BugTracker-2020` itself:


# 14:00:23 >  
# 14:00:23 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-76560" "--port=35661"
# 14:00:23 >  

#create procedure profiler.starttrace() external name profiler."starttrace";
#create procedure profiler.stoptrace() external name profiler.stoptrace;
#select 1 from (values ('one'), ('two')) as l(s), (values ('three'), ('four')) as r(s) where l.s like r.s;
% .%7 # table_name
% %7 # name
% tinyint # type
% 1 # length
#select count(*) from sys.tracelog() where stmt like '%cross%'; -- no crossproducts
% .%2 # table_name
% %2 # name
% bigint # type
% 1 # length
[ 0	]
#drop procedure profiler.starttrace();
#drop procedure profiler.stoptrace();
#select 1 from (values ('one'), ('two')) as l(s), (values ('three'), ('four')) as r(s) where l.s like r.s;
% .%7 # table_name
% %7 # name
% tinyint # type
% 1 # length

# 14:00:23 >  
# 14:00:23 >  "Done."
# 14:00:23 >  

